### PR TITLE
Cells-wrapper

### PR DESCRIPTION
**Pull request by madvingovich**

### CODE REVIEW

File: src/components/CellsWrapper.svelte [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<script>
+  import { onMount, onDestroy } from 'svelte';
+  export let total = 0;
+  export let getCellProps = null;
+  export let component = null;
+
+  let columns = '';
+
+  switch (true) {
```
Comment (dimkabuki) [inline src/components/CellsWrapper.svelte:9]: Yeah, that's looks not very flexible.
I would consider some coefficient instead

**Comment by dimkabuki** [comment]
Looks better, however, there is some gap from the right side?

**Comment by madvingovich** [comment]
> Looks better, however, there is some gap from the right side?

It works pretty quickly and we don't need a function to calculate a height
I'm not sure about svg, because we wanna render a component with props and it's hard to place elements in a grid with help of svg

**Comment by dimkabuki** [comment]
> > Looks better, however, there is some gap from the right side?
> 
> It works pretty quickly and we don't need a function to calculate a height
> I'm not sure about svg, because we wanna render a component with props and it's hard to place elements in a grid with help of svg

It's fine, I think we can go with it. It's quick and we can make it even quicker with pushing to gpu with css hack )
Just make it in the center or better spreading
I even think it could be implemented with the flex box and percentage to parent container

**Pull request merged**